torchtitan/experiments/simple_fsdp/parallelize.py

```diff
@@ -100,7 +100,22 @@ def parallelize_llama(
         logger.info("Applied Data Parallel (dp mode=%s) to the model", dp_mode)
 
     if job_config.training.compile:
+        from torch._inductor.comms import (
```

Comment:
nit: you don't need to import. you can pass string names of the passes instead, e.g. passes = ["sink_waits_iterative", ...]